Assets/Scripts/Models/Functions/FunctionsManager.cs

```diff
@@ -112,6 +112,9 @@ public static void JobComplete_FurnitureBuilding(Job theJob)
     {
         WorldController.Instance.World.PlaceFurniture(theJob.JobObjectType, theJob.tile);
 
+        // Let our workspot tile know it is no longer reserved for us
+        WorldController.Instance.World.UnreserveTileAsWorkSpot((Furniture)theJob.buildablePrototype, theJob.tile);
```

Comment:
No, the prototype isn't the same furniture as the instance of the furniture, this is removing the prototype, one using the furniture is added when the actual furniture is placed.